RecoMET/METProducers/src/HcalNoiseInfoProducer.cc

```diff
@@ -452,15 +696,50 @@ HcalNoiseInfoProducer::filldigis(edm::Event& iEvent, const edm::EventSetup& iSet
         coder.adc2fC(*digi, tool);
 
         for(int i = 0; i < (int)digi->size(); i++)
-           TotalCalibCharge = TotalCalibCharge + tool[i];
+           totalCalibCharge = totalCalibCharge + tool[i];
         */
```

Comment:
There is a large chunk of commented out code here above: couldn't it be just removed?
Otherwise, please add some further comment which explains why do you want to keep this commented out code here

Reply:
The current comment, 

   // Original code computes total calib charge over all digis.  While I think it would be more useful to skip
        // zs mark-and-pass channels, I keep this computation as is.  Individual HBHE and HF variables do skip
        // the m-p channels.  -- Jeff Temple, 6 December 2012

indicates that its quite old.   I can remove the commented code if desired

Reply:
Yes, please. Unless there is a reason to maintain it there (which I don't think)